sdk/ai/ai-agents/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.1.0-beta.2 (2025-07-18)
+
+### Features Added
+
+- Add MCP tool
+
+### Bugs Fixed
+
+- Add back `MicrosoftFabricTool`, `SharepointTool` and `BingCustomSearchTool` relate utils
```

Comment:
Why do we have this entry? I thought these tools have already been added in v1.1.0-beta.1.

Reply:
never mind, I see there are places where we missed adding them.